gensrc/thrift/FrontendService.thrift

```diff
@@ -531,6 +531,22 @@ struct TLoadTxnRollbackResult {
     1: required Status.TStatus status
 }
 
+struct TRLTaskCommitRequest {
+    1: required string taskId
+    2: required i64 signature
+    3: required i64 backendId
```

Comment:
why backendId is **required**?

Reply:
I need to remove task for `AgentTaskQueue`. And the params of removeTask need backendId